mautic/mautic
# Upgrade filemanager

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | 3.2 for bug fixes
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | no
| Related user documentation PR URL      | -
| Related developer documentation PR URL | -
| Issue(s) addressed                     | -

#### Description:
Fix critical bug in ELFinder with upgrade

 * Updated Composer package to studio-42/elfinder to 2.1.57  
 * Updated hardcoded assets in CoreBundle

![Snímek obrazovky 2020-11-24 v 11 33 16](https://user-images.githubusercontent.com/12815758/100083248-7841ca80-2e49-11eb-9b10-a07d5e67c335.png)

Current package versions: 
```
barryvdh/elfinder-flysystem-driver        v0.2.1                   
components/elfinder                       3.1.3                    
helios-ag/fm-elfinder-bundle              9.3.2                    
studio-42/elfinder                        2.1.57      
```
#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Create new segment e-mail
3. In builder click image icon and then filemanager
![Screen Shot 2020-11-18 at 1 18 50 PM](https://user-images.githubusercontent.com/12815758/99532820-30ccc180-29a5-11eb-8810-76035634e4e3.png)
4. Check ELFinder version under questionmark icon
5. Use filemanager features
